src/libraries/Microsoft.Extensions.Hosting/src/Microsoft.Extensions.Hosting.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFrameworks>netstandard2.0;netstandard2.1;net461</TargetFrameworks>
     <EnableDefaultItems>true</EnableDefaultItems>
     <PackageDescription>Hosting and startup infrastructures for applications.</PackageDescription>
+    <NoWarn>$(NoWarn);CP0001</NoWarn>
```

Comment:
Maybe we should have a granular way of doing NoWarn for Baseline Validation and regular validation?

Reply:
Yes this should be changed entirely,  I am currently adding this to have some validation enabled on these packages, and it will act as an end to end test for our changes in the msbuild sdk.